nservicebus/hosting/service-fabric-hosting/hosting-gateway.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Hosting Gateway with Service Fabric
+related:
+ - nservicebus/service-fabric
+reviewed: 2017-03-30
+---
+
+When adopting Service Fabric, it's not uncommon that the Service Fabric hosted endpoints need to interact with endpoints outside of the cluster. This can get tricky especially when the endpoints inside Service Fabric are stateful. When integrating using client side distribution, or when using the Service Fabric built in [reverse proxy](https://docs.microsoft.com/nl-nl/azure/service-fabric/service-fabric-reverseproxy) to expose the endpoints as web services, then the partition information needs to be provided by the consumer, which is often not desired.
```

Comment:
I'm on the fence here. I'm not sure we want to suggest using `Gateway` as a substitute for Client-Side Routing. I think we should tell users to `Gateway` for the purposes it was designed - which I think are still valid for FS